src/openms/source/METADATA/MetaInfo.cpp

```diff
@@ -113,27 +113,17 @@ namespace OpenMS
 
   bool MetaInfo::exists(const String & name) const
   {
-    try
+    UInt index = registry_.getIndex(name);
+    if (index != UInt(-1))
```

Comment:
I think that might give a "comparing signed to unsigned" warning, but I haven't tried it.